etl/steps/data/garden/missing_data/2024-03-14/children_out_of_school.meta.yml

```diff
@@ -0,0 +1,33 @@
+definitions:
+  common:
```

Comment:
Maybe it's useful to add a `description_processing` field to explain how the data has major processing.

Reply:
thank you! goood point!